sentry/src/test/java/io/sentry/DuplicateEventDetectionEventProcessorTest.kt

```diff
@@ -9,10 +9,22 @@ import kotlin.test.assertNull
 
 class DuplicateEventDetectionEventProcessorTest {
 
-    val processor = DuplicateEventDetectionEventProcessor(SentryOptions())
+    class Fixture {
+        fun getSut(enableDeduplication: Boolean? = null): DuplicateEventDetectionEventProcessor {
+            val options = SentryOptions().apply {
+                if (enableDeduplication != null) {
+                    this.setEnableDeduplication(enableDeduplication)
+                }
+            }
```

Comment:
you could init `SentryOptions` outside of `getSut` but still inside of `Fixture` and call options setters directly on each test, I think that's the way we've been doing so far.
`getSut` only takes params if the Sut (eg DuplicateEventDetectionEventProcessor) itself requires those params